testing/test_cacheprovider.py

```diff
@@ -793,9 +818,9 @@ def test_lastfailed_with_known_failures_not_being_selected(self, testdir):
         result = testdir.runpytest("--lf")
         result.stdout.fnmatch_lines(
             [
-                "collected 1 item",
-                "run-last-failure: 1 known failures not in selected tests (skipped 1 file)",
-                "* 1 failed in *",
+                "collected 2 items",
+                "run-last-failure: 1 known failures not in selected tests",
+                "* 1 failed, 1 passed in *",
```

Comment:
Not really sure, but this might show the fix for https://github.com/pytest-dev/pytest/issues/5301.
Could use a more explicit test otherwise, of course.